tests/unit/ui/components/TestRating.js

```diff
@@ -188,6 +188,11 @@ describe(__filename, () => {
     expect(findDOMNode(root).textContent).toEqual('Click to rate this add-on');
   });
 
+  it('renders nothing if no rating and readOnly', () => {
```

Comment:
Now you need to change the description of this test. Sorry.

Reply:
Whoops, right, heh 😄